proto/neutron/dex/tx.proto

```diff
@@ -105,11 +105,8 @@ message MsgPlaceLimitOrder {
       (gogoproto.jsontag) = "amount_in"
   ];
   LimitOrderType order_type = 8;
-  // expirationTime is only valid iff orderType == GOOD_TIL_TIME.
-  google.protobuf.Timestamp expiration_time = 9 [
-                                             (gogoproto.stdtime) = true,
-                                             (gogoproto.nullable) = true
-                                             ];
+  // Unix time in seconds. Expiration_time is only valid iff orderType == GOOD_TIL_TIME.
+  int64 expiration_time = 9;
```

Comment:
as a`google.protobuf.Timestamp` it was `(gogoproto.nullable) = true`, but as an `int64` now it's not marked as optional. I also can see that on the CLI side it's optional. so just to make sure, it's treated as optional with the `0` value passed as none, right?

I need to be sure on such things because I'm developing stargate helpers to be used on smart contracts' side and such things should be obvious for users. I think I'll also ask you to review optionality of request/response fields one day :) for now I simply rely on the `gogoproto.nullable` tags and mark the `true` ones as optional, will double check it against cli commands, but as the last check I'll need your help

Reply:
also: you sure about the decision of using int64 here instead of a uint? you can compare and cast between uint and int. I only don't understand why you use negative timestamps in tests